### PR TITLE
Updates for jitPack #bo-2093

### PR DESCRIPTION
**Pull request by jramos-spanning**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -76,7 +76,6 @@
         <!-- Eliminates the file encoding warning. Of course, all of your files
             should probably be UTF-8 nowadays. -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
Comment (jramos-spanning) [inline pom.xml:78]: Remove this so it will build on JitPack.

File: readme.md [hunk 2]
```diff
@@ -6,15 +6,35 @@
 
 This project is a fork of the Ms Office EWS api
 There are a few code changes.
```
Comment (jramos-spanning) [inline readme.md:8]: Update this for JitPack.

File: readme.md [hunk 3]
```diff
@@ -6,15 +6,35 @@
...
+4. Now, you can depend on the ews-java-api module by using `com.github.SpanningCloudApps` as the `groupId` and `ews-java-api`
+as the `artifactId`. For `version`, use `tagName` to depend on the a tag/release.
+
+  ```xml
+  <dependency>
+      <groupId>com.github.SpanningCloudApps</groupId>
+      <artifactId>ews-java-api</artifactId>
+      <version>2.1.2</version>
+  </dependency>
+  ```
```
Comment (vdietsch) [inline readme.md]: this is a public repo, so the above description is probably invalid for this project.  Can we double-check that we understand this before going forward with these harder-than-necessary instructions?

File: .idea/compiler.xml [hunk 4]
```diff
@@ -26,7 +26,7 @@
       </profile>
     </annotationProcessing>
     <bytecodeTargetLevel target="1.8">
-      <module name="ews-java-api" target="1.8" />
+      <module name="ews-java-api" target="1.7" />
```
Comment (jramos-spanning) [inline .idea/compiler.xml:29]: Didn't realize these IntelliJ files were checked in. I can revert this if anyone thinks we should/need to.

File: pom.xml [hunk 5]
```diff
@@ -76,6 +76,7 @@
...
+        <javaLanguage.version>1.7</javaLanguage.version>
```
Comment (jramos-spanning) [inline pom.xml:79]: Above comment is from the previous commit. In this commit I add it back in with a 1.7 version.

File: readme.md [hunk 6]
```diff
@@ -7,35 +7,6 @@
...
-as the `artifactId`. For `version`, use `tagName` to depend on the a tag/release.
-
-  ```xml
-  <dependency>
-      <groupId>com.github.SpanningCloudApps</groupId>
-      <artifactId>ews-java-api</artifactId>
-      <version>2.1.2</version>
-  </dependency>
-  ```
-
```
Comment (jramos-spanning) [inline readme.md:18]: Removed these Spanning specific instructions. The Cloudbees stuff is gone now too.

File: .idea/compiler.xml (hunk 4 above)
Comment (paulhernandez) [inline .idea/compiler.xml:29]: I didn't know we had these either.

File: readme.md (hunk 6 above)
Comment (paulhernandez) [inline readme.md:18]: :beer:

File: pom.xml (hunk 5 above)
Comment (vdietsch) [inline pom.xml:79]: Github's designers strike again!

File: readme.md (hunk 6 above)
Comment (vdietsch) [inline readme.md:18]: yep

**Pull request merged**